cpp/daal/src/algorithms/dtrees/gbt/regression/oneapi/gbt_regression_train_dense_default_oneapi_impl.i

```diff
@@ -732,7 +814,7 @@ services::Status RegressionTrainBatchKernelOneAPI<algorithmFPType, method>::comp
     auto & context        = Environment::getInstance()->getDefaultExecutionContext();
     auto & kernel_factory = context.getClKernelFactory();
 
-    __buildProgram<algorithmFPType>(kernel_factory);
+    DAAL_CHECK_STATUS_VAR(__buildProgram<algorithmFPType>(kernel_factory));
 
     kernelScan                        = kernel_factory.getKernel("scan", &status);
     kernelReduce                      = kernel_factory.getKernel("reduce", &status);
```

Comment:
Status check after getKernel calls

Reply:
There is one check after all getKernel calls below.